src/sonic_ax_impl/mibs/vendor/cisco/ciscoPfcExtMIB.py

```diff
@@ -0,0 +1,263 @@
+from enum import unique, Enum
+from bisect import bisect_right
+
+from sonic_ax_impl import mibs
+from ax_interface import MIBMeta, ValueType, MIBUpdater, MIBEntry, SubtreeMIBEntry
+from ax_interface.encodings import ObjectIdentifier
+
+class PfcUpdater(MIBUpdater):
+
+    def reinit_data(self):
+        """
+        Subclass update interface information
+        """
+        self.if_name_map, \
+        self.if_alias_map, \
+        self.if_id_map, \
+        self.oid_sai_map, \
+        self.oid_name_map = mibs.init_sync_d_interface_tables(self.db_conn)
+
+    def update_data(self):
+        """
+        Update redis (caches config)
+        Pulls the table references for each interface.
+        """
+        self.reinit_data()
```

Comment:
>reinit_data [](start = 13, length = 11)

reinit_data() is designed to run less frequent than update_data(). Why you need this call?

Reply:
I need to reinit_data because the port configuration could be changed and queues can be created/removed.  
For example, port break out feature splits the port to the new ones.

Reply:
@qiluo-msft 
Please, refer to the following flow of use case:
syncd switch init -> syncd set port/queues counters to counter DB -> snmp agent reads the counters and map from DB -> swss performs port break out -> syncd creates/removes the port and updates port and queue counters in Counter DB -> snmp agent reads updated map and values.

Reply:
In case we need to reinit data with low frequency, please specify what is expected frequency for reinit? 
Currently, it's updating every 5sec.

Reply:
@qiluo-msft 
Thanks a lot for the information. 
In this case I will refactor the update_data() to avoid the reinit.

Reply:
Fixed